projects/packages/masterbar/src/admin-color-schemes/colors/_overrides.scss

```
@@ -15,6 +15,7 @@
 
 	#wpadminbar{
 		background: $masterbar-background !important;
+		box-shadow: inset 0 -1px 0 $masterbar-background; // Calypso --color-masterbar-background
```

Maybe keep the comment from below?
```suggestion

		// Nav Unification - Masterbar - border below the Masterbar
		box-shadow: inset 0 -1px 0 $masterbar-background; // Calypso --color-masterbar-background
```

---

I figured in this case it was self-explanatory, but doesn't hurt. 🤷